wsmaster/che-core-sql-schema/src/main/resources/che-schema/7.16.0/2__update_che_workspace_activity_add_hard_time_limit.sql

```diff
@@ -0,0 +1,13 @@
+--
+-- Copyright (c) 2012-2020 Red Hat, Inc.
+-- This program and the accompanying materials are made
+-- available under the terms of the Eclipse Public License 2.0
+-- which is available at https://www.eclipse.org/legal/epl-2.0/
+--
+-- SPDX-License-Identifier: EPL-2.0
+--
+-- Contributors:
+--   Red Hat, Inc. - initial API and implementation
+--
+
+ALTER TABLE che_workspace_activity ADD COLUMN workspace_hard_expiration int8 null;
```

Comment:
if we decided to keep this field then we need to think about extending `CREATE INDEX che_index_ws_activity_expiration ON che_workspace_activity (expiration);` to make `WorkspaceActivity.getExpired` query efficient.